packages/optimizer-docker/index.js

```diff
@@ -2,8 +2,14 @@ const http = require('http');
 const url = require('url');
 const AmpOptimizer = require('@ampproject/toolbox-optimizer');
 const {parseRequest, getStaticOptions} = require('./utils');
+const {profiler, managementServer} = require('./metrics');
 
-const ampOptimizer = AmpOptimizer.create(getStaticOptions());
+// enable the profiling by default
+const configuration = Object.assign({profile: true, profiler}, getStaticOptions());
```

Comment:
Can you move this logic to the `getStaticOptions` function?
The main determinant for a "static" option is one that is set on init and is not modified on a per-request basis.